From 7868855563742a8f27f5dff022ca7f2e35aa28e0 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Tue, 23 Apr 2024 15:24:49 +0200
Subject: [PATCH 597/841] fix(dt-bindings): correct LSEDRV values

LSEDRV HIGH and MEDIUM values are inverted on SoC, align that in code.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 1c001e73d16f75834e58125e8ecb2d5a704f5c3e fix(dt-bindings): correct LSEDRV values]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ibc09dcaa782d0f0babc0aa7e4e5c017edcd6eb3e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/375826
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Gabriel FERNANDEZ <gabriel.fernandez@foss.st.com>
Reviewed-by: Gabriel FERNANDEZ <gabriel.fernandez@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 include/dt-bindings/clock/stm32mp25-clksrc.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/dt-bindings/clock/stm32mp25-clksrc.h b/include/dt-bindings/clock/stm32mp25-clksrc.h
index e6f7154b7..44c252403 100644
--- a/include/dt-bindings/clock/stm32mp25-clksrc.h
+++ b/include/dt-bindings/clock/stm32mp25-clksrc.h
@@ -219,8 +219,8 @@
 
 /* define for st,drive */
 #define LSEDRV_LOWEST		0
-#define LSEDRV_MEDIUM_LOW	1
-#define LSEDRV_MEDIUM_HIGH	2
+#define LSEDRV_MEDIUM_LOW	2
+#define LSEDRV_MEDIUM_HIGH	1
 #define LSEDRV_HIGHEST		3
 
 #endif /* _DT_BINDINGS_CLOCK_STM32MP25_CLKSRC_H_ */
-- 
2.47.2

